hr_timesheet_purchase_order/models/hr_timesheet_recurrence.py

```diff
@@ -106,7 +106,7 @@ class HRTimeSheetRecurrence(models.Model):
     sat = fields.Boolean(string="Sat")
     sun = fields.Boolean(string="Sun")
 
-    repeat_day = fields.Selection([(str(i), str(i)) for i in range(1, 32)])
+    repeat_day = fields.Integer()
```

Comment:
I think for this field need validation too.

Reply:
ok, I'll add validation for this field